src/dotnet-watch/Program.cs

```diff
@@ -16,17 +16,21 @@ public class Program : IDisposable
     {
         private readonly IConsole _console;
         private readonly string _workingDir;
-        private readonly CancellationTokenSource _cts;
         private IReporter _reporter;
 
+        // Use this token to check if the user has pressed CTRL+C
+        private readonly CancellationTokenSource _userInterrupt = new CancellationTokenSource();
```

Comment:
Does this ever get disposed?

Reply:
No. Should it?

Reply:
I generally try to and you do for _cts, so it's probably better to be consistent. It matters mostly if you creating CTS doesn't get canceled prior to being dereferenced in order to clean up the registrations.